src/renderer/screens/Assets/Assets.tsx

```diff
@@ -135,17 +132,18 @@ const Balances = () => {
           {accountIds.length > 0 && (
             <ul className="flex flex-col gap-y-4 items-center w-full py-4">
               {sortedChains.map((chain) => (
-                <NetworkBalances
-                  key={chain.chainId}
-                  hideZeroBalance={hideZeroBalance}
-                  searchSymbolOnly={searchSymbolOnly}
-                  query={query.toLowerCase()}
-                  chain={chain}
-                  accountIds={accountIds}
-                  canMakeActions={checkCanMakeActions()}
-                  onReceiveClick={onReceive(chain)}
-                  onTransferClick={onTransfer(chain)}
-                />
+                <li key={chain.chainId} className="w-[546px] mx-auto">
```

Comment:
Retrieved `<li>` from `NetworkAssets` so child component could be independent from environment